packages/composites/reference-transform/src/index.ts

```diff
@@ -1,9 +1,9 @@
 import { expose } from '@chainlink/ea-bootstrap'
 import { makeExecute } from './adapter'
-import { makeConfig } from './config'
+import { makeConfig, NAME } from './config'
 
-const NAME = 'REFERENCE_TRANSFORM'
+const adapterContext = { name: NAME }
 
-const handlers = expose(NAME, makeExecute())
+const handlers = expose(adapterContext, makeExecute())
```

Comment:
Since we are changing this line, could we modify this to be standard with a standard export style?
```suggestion
const { server } = expose(adapterContext, makeExecute())
```